src/style.css

```
@@ -0,0 +1,86 @@
+
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-weight: bold;
+}
+
+input {
+  font-family: Avenir, Helvetica, sans-serif;
```

move font-family to the `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```